portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -59,14 +59,12 @@ private void addEntity(String text, long timestamp, String email)
             {
                 break;
             }
-            else
-            {
-                String email = (String) entity.getProperty("email");
-                String eachComment = (String) entity.getProperty("commentInput");
-                String combine = email + ": " + eachComment;
-                cArray.add(combine);
-                count++;
-            }
+            String email = (String) entity.getProperty("email");
```

Comment:
I also commented on logic flattening by removing else with {}.

Reply:
I am not sure what you meant by logic flattening, but I removed the else statement to just have
if (numComments == count)
            {
                break;
            }
...